File: fizzbuzz.f90
program fizzbuzz
  implicit none
  integer i
  do i = 1, 100
    if (mod(i, 3) == 0 .and. mod(i, 5) == 0) then
      print *, "Fizz,Buzz"
    else if (mod(i, 3) == 0) then
      print *, "Fizz"
    else if (mod(i, 5) == 0) then
      print *, "Buzz"
    else
      print *, i
    end if
  end do
end program fizzbuzz

File: fizzbuzz.vb
Sub fizzbuzz()
　　Dim i As Long
　　Cells.Clear
　　For i = 1 To 100
　　　　If i Mod 15 = 0 Then
　　　　　　Cells(i, 4) = "FizzBuzz"
　　　　ElseIf i Mod 3 = 0 Then
　　　　　　Cells(i, 2) = "Fizz"
　　　　ElseIf i Mod 5 = 0 Then
　　　　　　Cells(i, 3) = "Buzz"
　　　　Else
　　　　　　Cells(i, 1) = i
　　　　End If
　　Next
End Sub
